src/SmartComponents/DriftPage/DriftPage.js

```diff
@@ -29,4 +34,10 @@ class DriftPage extends Component {
     }
 }
 
+DriftPage.propTypes = {
+    getFilterByState: PropTypes.func,
+    filter: PropTypes.string,
+    stateFilter: PropTypes.string
+};
```

Comment:
I don't see these props being used anywhere.

Reply:
ah yes, you're right. I forgot to remove them while I was trying things out.